pkg/apis/crd/v1alpha2/types.go

```diff
@@ -389,3 +389,19 @@ type IPPoolList struct {
 
 	Items []IPPool `json:"items"`
 }
+
+// TODO: remove this after #3487 is merged.
+type Direction string
+
+const (
+	DirectionIn   Direction = "In"
+	DirectionOut  Direction = "Out"
+	DirectionBoth Direction = "Both"
+)
+
+type Action string
```

Comment:
This package is shared by all CRDs. We need to add some prefix, like TrafficControlAction

Reply:
I'll rebase this part with PR #3644